src/pykeen/pipeline.py

```diff
@@ -699,6 +699,34 @@ def pipeline_from_config(
     )
 
 
+def save_pipeline_checkpoint_helper_file(path: str, random_seed: int) -> None:
+    """Save the pipeline checkpoint helper file.
+
+    :param path:
+        Save the state of the pipeline.
+    :param random_seed:
+        The random_seed that was used for the pipeline.
+    """
+    torch.save(
+        {
+            'random_seed': random_seed,
+        },
+        path,
+    )
+
+
+def load_pipeline_checkpoint_helper_file(path: str) -> Mapping[str, Any]:
```

Comment:
does this really need its own function?

Reply:
I agree that it's a bit overkill right now, but it was build with expandability in mind for the entire pipeline and it also adds some clarity to the code about what these files actually contain.
But for now I would also be finde with only making the straight function calls without the helper functions.

Reply:
I think there's an old rule for software dev not to implement new stuff until you really need it. Another way we could go about this is to use TypedDicts / type annotations to add into the code directly where torch.load is called